cranelift/isle/docs/language-reference.md

```diff
@@ -26,7 +26,7 @@ instructions. For example:
   be lowered to an x86 `ADD` instruction with a register and an
   immediate.
 
-One could write something like the following in ISLE (simplified from
+One can write something like the following in ISLE (simplified from
```

Comment:
This rewrite loses the meaning: the whole stanza is of the form "one could ... [but] ISLE lets you ..." and changing "could" to "can" completely changes the meaning.